### PR TITLE
Brendan's portion

### PR DESCRIPTION
**Pull request by myhe01**

### CODE REVIEW

File: control.tex [hunk 1]
```diff
@@ -0,0 +1,61 @@
+\section{Controller Subsystem}
+
+% Why the MCU connects to the internet
+In order for our system to be as self and power efficient as possible from an end-user perspective, our team decided to use a low-power, Internet-of-Things (IoT)-focused wireless microcontroller. To make the process of operating our product as hands-off as possible to end-users, the microcontroller will operate in Access Point (AP) mode to serve information to the user's mobile device. We do not expect our product to produce or receive large amounts of data, so the decreased bandwidth of a WiFi-enabled product being beyond the outdoor walls of a building is not a significant drawback to our application.
+
+% How the MCU connects to the internet (local network, LAN -> NAT -> WAN, TCP stack)
+\paragraph{CC3220 Overview}
```
Comment (NChitty) [inline control.tex:7]:
```suggestion
\subsection{CC3220 Overview}
```

**Review by NChitty — DISMISSED** [review]
Replace `\paragraph` with `\subsection`

I also based on reading other papers [(1)](https://webcourses.ucf.edu/courses/1421149/files/folder/EOS_Spring_2023?preview=97156045) and [(2)](https://webcourses.ucf.edu/courses/1421149/files/folder/EOS_Spring_2023?preview=97156042) I don't think we need a pure summary of the design document because right now it just reads like a lot of fluff. I think we can in the conference paper focus a lot more on implementation details such as pin config due to pinmux or transistors to avoid frying the board, or timing stuff. You have done a lot with the control subsystem and I don't think this iteration does it justice.

**Comment by myhe01** [comment]
Please rereview.

**Comment by myhe01** [comment]
Pulled main into this branch

**Review by NChitty — APPROVED** [review]

**Pull request merged**